wordpress-mobile/WordPress-iOS
# Adds more support for Dynamic Type

Fixes #7068 

This PR adds support for dynamic type in the following places:

 My Sites
 Blog Header
 Blog Table
 Blog -> Pages
 Blog -> Comments
 Blog -> Sharing
 Blog -> Settings
 Me
 Me Header
 Me->My Profile
 Me->Account Settings
 Me->App Settings
 Me->Notifications
 Post Options
 Post Options -> Categories
 Post Options -> Tags
 Post Options -> Publicize Message/Slug/Excerpt
 Search Bar
 Reader Comments
 Labels on the navigation bar
 Reply View
 Filter on Post List

At this stage we want to improve on what we currently had (many things broke on large text sizes) and provide at support for lager text on the most visible text in the app.

@mattmiklic as in the previous PR https://github.com/wordpress-mobile/WordPress-iOS/pull/7201 some font sizes had to change slightly. A sample here:

| Develop        | This branch           |
| ------------- |-------------|
|![medevelop](https://cloud.githubusercontent.com/assets/5558824/26375910/fe2f67a2-3fe0-11e7-8d31-4fb067022d82.png) |![me](https://cloud.githubusercontent.com/assets/5558824/26375909/fdef4de8-3fe0-11e7-87cc-bb1f8551e2ef.png)|
|![postoptionsdevelop](https://cloud.githubusercontent.com/assets/5558824/26374618/48b38880-3fdc-11e7-816b-a1de72e44b24.png) |![postoptions](https://cloud.githubusercontent.com/assets/5558824/26374478/da20f254-3fdb-11e7-8c08-25511fc6493d.png)|
|![blogdetailsdevelop](https://cloud.githubusercontent.com/assets/5558824/26374617/48ae6b84-3fdc-11e7-9416-77c76b9f5fa2.png)|![blotdetails](https://cloud.githubusercontent.com/assets/5558824/26374479/da29dc84-3fdb-11e7-97d7-da47c31eae6b.png)|


To test:
Enable "Larger Text" on Settings->General->Accessibility->Larger Text
Check out all the places listed above.

Needs review: @nheagy
